.gitignore

```diff
@@ -119,3 +119,6 @@ version.py
 
 # Ape stuff
 .build/
+
+# VS Code
```

Comment:
I was told that IDE specific entries should be put in our global `.gitignore` files

Reply:
oops